salt/modules/win_pkg.py

```diff
@@ -695,11 +696,13 @@ def install(name=None, refresh=False, pkgs=None, saltenv='base', **kwargs):
     new = list_pkgs()
     tries = 0
     difference = salt.utils.compare_dicts(old, new)
-    while not all(name in difference for name in changed) and tries <= 1000:
+    while not all(name in difference for name in changed) and tries < 10:
```

Comment:
Hmmm I think that should be ok: 
 - `tries = 9`
 - `9 < 10`; so evaluates to `True` and enters `while` statement
 - `tries += 1` so `tries = 10`
 - `10 == 10` to evaluates to `True` and enters `if` statement